### PR TITLE
Updating disposition_ids to match firewall profile

### PR DESCRIPTION
**Pull request by adplotzk**
Updating disposition_ids to match firewall profile.

This is in response to the PR https://github.com/ocsf/ocsf-schema/pull/732.

### CODE REVIEW

**Review by floydtree — APPROVED** [review]

**Review by mr-splunk — COMMENTED** [review]
Something I noticed with Palo Alto logs - along with `blocked` there is also a `dropped`. I recall that there is an important distinction between the two. Should `dropped` be worth considering as a disposition?

**Comment by adplotzk** [comment]
> Something I noticed with Palo Alto logs - along with `blocked` there is also a `dropped`. I recall that there is an important distinction between the two. Should `dropped` be worth considering as a disposition?
> 
> image

I can add this.

**Comment by adplotzk** [comment]
> Something I noticed with Palo Alto logs - along with `blocked` there is also a `dropped`. I recall that there is an important distinction between the two. Should `dropped` be worth considering as a disposition?
> 
> image

I've added some additional Palo actions to the disp_ids. can you please review again.

File: profiles/firewall.json [hunk 1]
```diff
@@ -17,62 +17,36 @@
...
-        "13": {
-          "caption": "Uncorrected"
-        },
-        "14": {
-          "caption": "Delayed",
-          "description": "Requires reboot to finish the operation."
-        },
-        "15": {
-          "caption": "Detected"
+        "3": {
```
Comment (rroupski) [inline profiles/firewall.json:35]: I see duplicate enum value id's. Can you fix them?

**Review by rroupski — CHANGES_REQUESTED** [review]

**Review by rroupski — APPROVED** [review]

**Review by floydtree — APPROVED** [review]

**Review by Aniak5 — APPROVED** [review]
LGTM!

**Pull request merged**